docs/release_notes/next/fix-2422-Add-roi-after-export

```diff
@@ -0,0 +1 @@
+2422: Spectrum viewer: cant add an ROI after exporting CSV
```

Comment:
Can we re-word this slightly to say what the proposed change is resolving, maybe something like: 

> "Spectrum viewer: Resolves a bug preventing a new ROI from being added  after exporting to a CSV file,"